sdk/tables/azure-data-tables/test/ut/table_client_test.cpp

```diff
@@ -674,19 +644,12 @@ namespace Azure { namespace Data { namespace Test {
         case AuthType::Key:
           stringValue = "key";
           break;
-        case AuthType::SAS:
-          stringValue = "sas_LIVEONLY_";
-          break;
         default:
           stringValue = "key";
           break;
       }
       return stringValue;
     }
   } // namespace
-  INSTANTIATE_TEST_SUITE_P(
-      Tables,
-      TablesClientTest,
-      ::testing::Values(AuthType::Key, AuthType::SAS),
-      GetSuffix);
+  INSTANTIATE_TEST_SUITE_P(Tables, TablesClientTest, ::testing::Values(AuthType::Key), GetSuffix);
```

Comment:
Does this still need to be a parameterized test suite?

Or would that introduce too much churn?

Reply:
i left it there as it we might need to bring back sas, also yes it would bring too much churn for no functional benefit.